scripts/node/icons/helpers/cleanExports.js

```diff
@@ -14,32 +14,31 @@ const fs = require('fs');
 const path = require('path');
 const glob = require('glob');
 const rimraf = require('rimraf');
-const pascalCase = require('pascal-case');
+const { pascalCase } = require('pascal-case');
```

Comment:
This is the only thing needed to actually fix the bug. Everything else is around writing cleaner code, writing a test, and doing some experimenting while I'm doing that.